aips/passkeys.md

```diff
@@ -0,0 +1,504 @@
+
+## Specification
+
+### Terminology
+
+- [**OIDB account**](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-61.md): An OpenID blockchain (OIDB) account whose security and liveness is backed by an OIDC account (e.g., a Google account) rather than a secret key.
+- [**Relying party**](https://www.w3.org/TR/webauthn-3/#relying-party): The website, application, or service that the user is trying to access, which is responsible for verifying a signature from the user's passkey to ensure that the user is who they claim to be.
+- [**WebAuthn Authenticator**](https://www.w3.org/TR/webauthn-3/#authenticator): A cryptographic entity, existing in hardware or software, that can register a user with a given relying party and later assert possession of the registered public key credential, and optionally verify the user to the relying party.
+
+
+### Aptos Account Creation with Passkeys
+
+During the registration of a new WebAuthn credential, an asymmetric key (i.e., a **private key** and its associateed **public key**) is generated securely on that device. The public key associated with the WebAuthn credential can be used to derive an associated account on chain. The private key, which controls the account, will be secured by the device.
+
+The registration process is particularly important for several reasons:
+
+1. The public key associated with the credential is only revealed during registration. Assertion responses (signing with the passkey) do NOT return the public key. Thus, the registration response must be parsed and the relevant information about the credential should be stored properly and persisted in storage by the relying party.
+2. The registration response includes `flags` that describe the backup characteristics of the credential. If passkey backup is an important consideration, these flags should be evaluated **BEFORE** the on-chain account is created.
+
+#### [`PublicKeyCredentialCreationOptions`](https://www.w3.org/TR/webauthn/#dictdef-publickeycredentialcreationoptions)
+
+Every WebAuthn credential is created by the relying party (RP) with a set of options, called `PublicKeyCredentialCreationOptions`. These options help configure many aspects of the WebAuthn credential, including, but not limited to:
+
+- The asymmetric key type for the credential (`Ed25519`, `ECDSA`, `RSA`, etc.)
+- User verification requirements during assertions
+- Whether the WebAuthn credential should be bound to one device or available cross-platform. 
+
+It's important to carefully consider which options to select as they have significant impacts on the passkey user experience and **CANNOT** be reconfigured after credential creation.
+
+```
+dictionary PublicKeyCredentialCreationOptions {
```

Comment:
Not sure if this level of detail is necessary.

Reply:
Removed